src/main/java/appeng/util/InventoryAdaptor.java

```diff
@@ -67,10 +69,26 @@ public static InventoryAdaptor getAdaptor(final EntityPlayer te) {
         return null;
     }
 
-    // return what was extracted.
-    public abstract ItemStack removeItems(int amount, ItemStack filter, IInventoryDestination destination);
-
-    public abstract ItemStack simulateRemove(int amount, ItemStack filter, IInventoryDestination destination);
```

Comment:
Ditto.

Reply:
It looks like the new method signature just adds a boolean parameter. Consider setting a default value. to make other plugins compatible

Reply:
> Ditto.

It looks like the new method signature just adds a boolean parameter. Consider using overloading. to make other plugins compatible